app/decorators/controllers/accounts_controller_decorator.rb

```diff
@@ -81,12 +88,24 @@ def last_assignment_due
     account_settings_params[:last_assignment_due].present? ? account_settings_params[:last_assignment_due] : false
   end
 
+  def course_start_time_hour
+    account_settings_params[:course_start_time_hour].present? ? account_settings_params[:course_start_time_hour] : "00"
```

Comment:
if we're sticking with the 12 hour format, i think the default hour should be "12", right?

Reply:
Yes good catch, thank you. Made the change in the new PR.